From 6058ea70d19c70464f3190e0d3ed5cd64eb21631 Mon Sep 17 00:00:00 2001
From: Luca Di Maio <luca.dimaio1@gmail.com>
Date: Tue, 2 Dec 2025 16:11:10 +0100
Subject: [PATCH v0] fix: treat faillock as directory

-- in rules are for regular files, faillock is a dir.

Ref VMS-231

Signed-off-by: Luca Di Maio <luca.dimaio1@gmail.com>
---
 policy/modules/system/authlogin.fc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy/modules/system/authlogin.fc b/policy/modules/system/authlogin.fc
index 375777ab2..0adbc3c1b 100644
--- a/policy/modules/system/authlogin.fc
+++ b/policy/modules/system/authlogin.fc
@@ -60,7 +60,7 @@ ifdef(`distro_suse', `
 /var/log/btmp.*		--	gen_context(system_u:object_r:faillog_t,s0)
 /var/log/dmesg		--	gen_context(system_u:object_r:var_log_t,s0)
 /var/log/faillog.*	--	gen_context(system_u:object_r:faillog_t,s0)
-/var/log/faillock.*	--	gen_context(system_u:object_r:faillog_t,s0)
+/var/log/faillock.*		gen_context(system_u:object_r:faillog_t,s0)
 /var/log/lastlog.*	--	gen_context(system_u:object_r:lastlog_t,s0)
 /var/log/syslog		--	gen_context(system_u:object_r:var_log_t,s0)
 /var/log/tallylog.*	--	gen_context(system_u:object_r:faillog_t,s0)
-- 
2.52.0

